Doc/library/argparse.rst

```diff
@@ -426,7 +426,9 @@ should not be line-wrapped::
     -h, --help  show this help message and exit
 
 :class:`RawTextHelpFormatter` maintains whitespace for all sorts of help text,
-including argument descriptions.
+including argument descriptions. However, multiple new lines in epilog and
+description are replaced with one. If you prefer to use multiple new lines,
+add white space between the new line characters.
```

Comment:
I don't think we need to mention epilog and description here.  It sounds like *all* instances of adjacent newlines are turned into single newlines.

I would phrase the last sentence: "If you wish to preserve multiple blank lines, add spaces between the newlines.

Reply:
I've added the requested changes.